nilearn/plotting/cm.py

```diff
@@ -181,12 +181,6 @@ def alpha_cmap(color, name="", alpha_min=0.5, alpha_max=1.0):
     _cmaps_data["hot_white_bone"] = _concat_cmap(_cm.afmhot, _cm.bone_r)
     _cmaps_data["hot_black_bone"] = _concat_cmap(_cm.afmhot_r, _cm.bone)
 
-# Copied from matplotlib 1.2.0 for matplotlib 0.99 compatibility.
```

Comment:
probably better to have some DeprecationWarning and an alias to the matploblib bwr introduced in >=3.5